org.eclipse.scanning.test/src/org/eclipse/scanning/test/malcolm/AbstractAbortingMalcolmTest.java

```diff
@@ -22,6 +22,9 @@
 import org.junit.runner.RunWith;
 import org.junit.runners.Parameterized;
 
+/**
+ * TODO DAQ-1004 see comment in {@link AbstractMalcolmTest}
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)